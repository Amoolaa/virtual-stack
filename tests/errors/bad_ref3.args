"tests/errors/bad_ref3.x2017"

