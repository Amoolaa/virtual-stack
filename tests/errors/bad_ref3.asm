FUNC LABEL 0
    MOV STK A VAL 1
    MOV REG 0 STK A
    REF STK B REG 0
    PRINT PTR B
    RET
