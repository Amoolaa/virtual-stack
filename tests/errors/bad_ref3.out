REF: second argument must be a stack symbol.
